frontend/www/js/omegaup/components/course/AssignmentDetails.vue

```diff
@@ -331,7 +332,43 @@ export default class CourseAssignmentDetails extends Vue {
   }
 
   onSubmit(): void {
-    this.$emit('submit', this, this.scheduledProblemList?.problems ?? []);
+    let params = {
+      name: this.name,
+      description: this.description,
+      assignment_type: this.assignmentType,
+    };
+    if (this.unlimitedDuration) {
+      params = { ...params, ...{ unlimited_duration: true } };
```

Comment:
algo más conciso es

```suggestion
      params.unlimited_duration = true;
```

igual con todos los demás campos individuales que se agregan.

Reply:
Listo